src/gamecontroller.cs

```diff
@@ -1922,4 +1996,50 @@ enum GCInputDirectional {
 		[Field ("GCInputDirectionalCardinalDpad")]
 		CardinalDpad,
 	}
+
+	delegate GCVirtualControllerElementConfiguration GCVirtualControllerElementUpdateBlock (GCVirtualControllerElementConfiguration configuration);
+
+	[NoTV, NoMac, NoWatch, iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface GCVirtualController
+	{
+		[Static]
+		[Export ("virtualControllerWithConfiguration:")]
+		GCVirtualController CreateVirtualController (GCVirtualControllerConfiguration configuration);
+
+		[Export ("connectWithReply:")]
+		void Connect ([NullAllowed] Action<NSError> reply);
+
+		[Export ("disconnect")]
+		void Disconnect ();
+
+		[NullAllowed, Export ("controller", ArgumentSemantic.Weak)]
+		GCController Controller { get; }
+
+		[Export ("changeElement:configuration:")]
+		void Change (string element, GCVirtualControllerElementUpdateBlock config);
```

Comment:
This is minor, but we try to not use abbreviations for parameter names:

```suggestion
		void Change (string element, GCVirtualControllerElementUpdateBlock configuration);
```

Reply:
Ah got it!